contrib/container-chroot-agent/Dockerfile-cfengine-chroot-agent

```diff
@@ -0,0 +1,14 @@
+#+caption: Dockerfile for CFEngine Enterprise Agent
+#+begin_src
+FROM debian:11
```

Comment:
I used debian:11 instead of :12 or :latest because my torizon core system docker version couldn't handle anything newer.